src/cpp/rtps/builtin/data/ParticipantProxyData.hpp

```diff
@@ -76,6 +77,8 @@ class ParticipantProxyData
     GUID_t m_guid;
     //!Vendor ID
     fastdds::rtps::VendorId_t m_VendorId;
+    //! Product version
+    fastdds::rtps::ProductVersion_t product_version;
```

Comment:
I wonder if it would be a good idea to propagate product_version to other structures such as cache changes or writer/reader proxy datas, same as it's done with vendor_id.

Reply:
Vendor_is is propagate in `CacheChange_t` because that information comes in each RTPS message. Furthermore we don't think it is necessary at this moment.